docs/usage/cfcli/application_deployment.md

```diff
@@ -1,97 +1,29 @@
-## Application Deployment
+The command to create a new app and to push a new version of an existing one are the same: `cf push`.
 
-To get a list of existing apps depoloyed in the curent space run `cf apps`.
+Cloud Foundry isn’t version-control-aware. `cf push` will deploy the working state of whatever files you have in your current directory.
 
-The command used for deploying a new app, or for updating an existing app, is `cf push ...`
+Before deploying your application you should [setup a manifest.yml](#). The manifest.yml file tells `cf push` what to do. This includes everything from how many instances to create to how much memory or disk space to assign.
 
-There is a fair amount of options available to the `cf push` command. To see them all run `cf push --help`
+Another option is to define these configurations as options on `cf push`. Not defining configuration options at all may slow down your deployments.
```

Comment:
possibly change `options on cf push` to `options to cf push`. Also possibly `slow down your deployments or provide too many or too few resources for your application.` That sentence needs work but the manifest also specifies things like memory and disk allocation.